src/action-creators/loadRemoteState.js

```diff
@@ -64,7 +64,7 @@ export const loadState = (newState, oldState) => {
     // subthoughts.lastUpdated > oldSubthoughts.lastUpdated
     // technically subthoughts is a disparate list of ranked thought objects (as opposed to an intersection representing a single context), but equalPath works
     if (subthoughts && subthoughts.length > 0 && !equalPath(subthoughts, subthoughtsOld)) {
-      localForage.setItem('contextIndex-' + contextEncoded, subthoughts)
+      dbOperations.updateContextIndex(contextEncoded, subthoughts)
```

Comment:
How about batching all of the updates into one db call for better performance?

Reply:
That's a good idea. Let me try doing that